.gitignore

```diff
@@ -43,3 +43,4 @@ tags
 recipients_app/android/app/src/main/google-services.json
 recipients_app/android/app/google-services.json
 **/.cxx
+recipients_app/ios/Flutter/ephemeral
```

Comment:
@novas1r1 As far as I understand those ephemeral files, they get generated with every Flutter ios build and they can change during Flöutter versions. So I think it is better to not commit them to the repo.

Reply:
I deleted those files and git ignored them. Hope that is ok for you.